doc/submitting-changes.xml

```diff
@@ -224,6 +224,139 @@ Additional information.
 </section>
 
 <section>
+  <title>Pull Request Template</title>
+  <para>
+    The pull request template helps determine what steps have been made for a
+    contribution so far, and will help guide maintainers on the status of a
+    change. The motivation section of the PR should include any extra details
+    the title does not address and link any existing issues related to the pull
+    request.
+  </para>
+  <para>When a PR is created, it will be pre-populated with some checkboxes detailed below:
+  </para>
+  <section>
+    <title>Tested using sandboxing</title>
+    <para>
+      When sandbox builds are enabled, Nix will setup an isolated environment
+      for each build process. It is used to remove further hidden dependencies
+      set by the build environment to improve reproducibility. This includes
+      access to the network during the build outside of
+      <literal>fetch*</literal> functions and files outside the Nix store.
+      Depending on the operating system access to other resources are blocked
+      as well (ex. inter process communication is isolated on Linux); see <link
+      xlink:href="https://nixos.org/nix/manual/#description-45">build-use-sandbox</link>
+      in nix manual for details.
+    </para>
+    <para>
+      Sandboxes are not enabled by default in Nix as there are cases where it
+      makes building packages harder (for example <command>npm
+        install</command> will not work due missing network access). In pull
+      requests for <link
+        xlink:href="https://github.com/NixOS/nixpkgs/">nixpkgs</link> people
+      are asked to test builds with sandboxing enabled (see 
+      <literal>Tested using sandboxing</literal> in the pull request template)
+      because in<link xlink:href="https://nixos.org/hydra/">https://nixos.org/hydra/</link>
+      sandboxing is also used.
+    </para>
+    <para>
+      Depending if you use NixOS or other platforms you can use one of the
+      following methods to enable sandboxing:
+      <itemizedlist>
+        <listitem>
+          <para><emphasis role="bold">Enable sandboxing for single build</emphasis>:
+            <screen>nix-shell -I nixpkgs=/path/to/nixpkgs --option build-use-sandbox true -p hello</screen>
```

Comment:
The way this is worded gives the impression that an ordinary user can control sandbox usage even if they are using the Nix daemon.

Reply:
Are options ignored, when nix-daemon is used? Then we should probably not mention this at all.
Otherwise people, will fall into the trap.

Reply:
I agree it'd be better to leave this out, or at least add some caveats.